features/db.feature

```diff
@@ -21,6 +21,9 @@ Feature: Perform database operations
     When I try the previous command again
     Then the return code should be 1
 
+    When I run `wp db check`
+    Then STDOUT should not be empty
```

Comment:
Do you have security-lazy users??

![secure-wplogin](https://cloud.githubusercontent.com/assets/952007/17608186/f90d956e-602a-11e6-8bb4-36c11eaf6bf5.png)

It is free: https://github.com/szepeviktor/wordpress-plugin-construction/blob/master/mu-keepass-button/keepass-button.php